.travis.yml

```diff
@@ -35,6 +35,6 @@ install:
     - cd client && npm install && cd ..
 
 script:
-    - cd server && flake8 && cd ..
+    - cd server && flake8 && pytest && behave && cd ..
```

Comment:
Are we going to use pytest to run tests for fidelity?

Reply:
I've just enabled it, seems simpler